manifests/0000_10_config-operator_07_deployment.yaml

```diff
@@ -36,6 +36,26 @@ spec:
         secret:
           secretName: config-operator-serving-cert
           optional: true
+      initContainers:
+        - name: openshift-api
+          securityContext:
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop: ["ALL"]
+          terminationMessagePolicy: FallbackToLogsOnError
+          image: quay.io/openshift/origin-cluster-config-api:v4.0
+          imagePullPolicy: IfNotPresent
+          command:
+            - write-available-featuresets
+            - asset-output-dir=/available-featuregates
```

Comment:
Do you want to use an empty dir for this? And use that to share the manifests between the two containers?

Currently the E2E is failing on unable to `mkdir`